sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/AvroGenericRecordToStorageApiProto.java

```diff
@@ -262,8 +262,7 @@ private static TableFieldSchema fieldDescriptorFromAvroField(Schema.Field field)
             elementType.getType() != Schema.Type.UNION,
             "Multiple non-null union types are not supported.");
         TableFieldSchema unionFieldSchema =
-            fieldDescriptorFromAvroField(
-                new Schema.Field(field.name(), elementType, field.doc(), field.defaultVal()));
+            fieldDescriptorFromAvroField(new Schema.Field(field.name(), elementType, field.doc()));
```

Comment:
Is it going to be compiled successfully with previous Avro versions?

Reply:
In this case, the code here just is about trying to determine schema information. the default value doesn't appear to be needed